.husky/pre-commit

```diff
@@ -6,7 +6,7 @@ set -e
 echo 'Running pre-commit hook...'
 
 # check typos
-chronic typos .
+chronic typos --exclude core/packages/contracts/lib .
```

Comment:
https://github.com/Snowfork/snowbridge/blob/be58054456a890b46574d5a8483b520ac11b09ef/_typos.toml#L2

could also add into this file

Reply:
done